components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -210,8 +211,10 @@ func createGitpodService(cfg *Config, tknsrv api.TokenServiceServer) *gitpod.API
 		return nil
 	}
 	tknres, err := tknsrv.GetToken(context.Background(), &api.GetTokenRequest{
+		Kind: "gitpod",
```

Comment:
maybe we should have a few constants in `supervisor-api` for common token kinds.
E.g.

```
const (
    // KindGitpod marks tokens that provide access to the Gitpod server API
    KindGitpod = "gitpod"
    
    // KindGit marks any kind of Git access token
    KindGit = "git"
)
```

and something like that on the TS side if that makes sense (some uses that).